Firestore/Source/Remote/FSTRemoteEvent.mm

```diff
@@ -605,44 +541,91 @@ - (void)recordResponseForTargetID:(FSTBoxedTargetID *)targetID {
  * preventing in preventing race conditions for a target where events arrive but the server hasn't
  * yet acknowledged the intended change in state.
  */
-- (BOOL)isActiveTarget:(FSTBoxedTargetID *)targetID {
+- (BOOL)isActiveTarget:(FSTTargetID)targetID {
```

Comment:
What's the point of this change? This forces a caller that has a boxed target ID to unbox so it can be reboxed here.